DrivewayDentDeletion/Operators/prereqs.sh

```diff
@@ -76,90 +99,78 @@ oc adm policy add-scc-to-group privileged system:serviceaccounts:${test_namespac
 
 echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
-echo "INFO: Installing OCP pipelines"
-cat <<EOF | oc apply -f -
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  name: openshift-pipelines-operator
-  namespace: openshift-operators
-spec:
-  channel: ocp-4.4
-  name: openshift-pipelines-operator-rh
-  source: redhat-operators
-  sourceNamespace: openshift-marketplace
-EOF
-
-echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
-
-echo "Creating secrets to push images to openshift local registry"
-export DOCKER_REGISTRY="image-registry.openshift-image-registry.svc:5000"
-export username=image-bot
-kubectl -n ${dev_namespace} create serviceaccount image-bot
-oc -n ${dev_namespace} policy add-role-to-user registry-editor system:serviceaccount:${dev_namespace}:image-bot
+echo "INFO: Adding permission for '$dev_namespace' to write images to openshift local registry in the '$test_namespace'"
 # enable dev namespace to push to test namespace
 oc -n ${test_namespace} policy add-role-to-user registry-editor system:serviceaccount:${dev_namespace}:image-bot
```

Comment:
`image-bot` sounds like a general purpose service account and this permission is DDD specific. I guess this service account name should have `ddd` in it.